.github/workflows/wcf_backport_dispatch.yml

```diff
@@ -0,0 +1,181 @@
+          GH_DEBUG: ${{ runner.debug == '1' }}
+        run: |
+          if [[ "${{ runner.debug }}" == "1" ]]; then
+            set -x
+          fi
+          
+          # target_branches
+          if [[ "${{ inputs.target_branches }}" != "" ]]; then
+            TARGET_BRANCHES=$(echo "[\"${{ inputs.target_branches }}\"]" | sed 's/ //g' | sed 's/,/","/g')
+            echo "branches=${TARGET_BRANCHES}" >> $GITHUB_OUTPUT
+          else
+            # The head -1 is because GITHUB_OUTPUT is easier to work with single line output
+            TARGET_BRANCHES=$(gh api /repos/${{ github.repository }}/contents/${{ inputs.branchFilePath }} --jq '.content | @base64d' | jq  -cM '.[:index("${{ steps.get-pr-info.outputs.pr_base_ref }}")]')
+            echo "branches=${TARGET_BRANCHES}" >> $GITHUB_OUTPUT
+          fi
+  open-prs:
+    permissions:
+      contents: read  # for actions/checkout to fetch code, when use github-app-token with secrets set
+      # contents: write  # when use context token without secrets set
+      pull-requests: write  # for marocchino/sticky-pull-request-comment to create or update PR comment
+    needs:
+      - collect-info
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: ${{ fromJSON(needs.collect-info.outputs.branches) }}
+    runs-on: ubuntu-latest
+    env:
+      GH_USER: ${{ inputs.gitUser }}
+      GH_EMAIL: ${{ inputs.gitEmail }}
+      PR_NUMBER: ${{ inputs.pr_number }}
+      PR_TITLE: ${{ needs.collect-info.outputs.pr_title }}
+      SHA: ${{ needs.collect-info.outputs.pr_merge_commit_sha }}
+      TARGET_BRANCH: ${{ matrix.branch }}
+      USE_APP_TOKEN: ${{ secrets.app_id != '' }}
+    steps:
+      - uses: marocchino/sticky-pull-request-comment@efaaab3fd41a9c3de579aba759d2552635e590fd # v2.8.0
+        with:
+          append: true
+          message: backporting to ${{ matrix.branch }} with [action](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+      - id: github-app-token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        if: ${{ env.USE_APP_TOKEN == 'true' }}
+        with:
+          app_id: ${{ secrets.app_id }}
+          private_key: ${{ secrets.private_key }}
+      - id: github-context-token
+        if: ${{ env.USE_APP_TOKEN != 'true' }}
+        run: |
+          echo "token=${{ github.token }}" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: ${{ env.TARGET_BRANCH }}
+          fetch-depth: 0
+      - name: git-config
+        run: |
+          git config user.name "${GH_USER}"
+          git config user.email "${GH_EMAIL}"
+      - name: cherry-pick
+        run: |
+          if [[ "${{ runner.debug }}" == "1" ]]; then
+            set -x
+          fi
+          
+          echo LABELS=${{ matrix.branch }} >> $GITHUB_ENV
+          if git cherry-pick ${{ env.SHA }}; then
```

Comment:
Bakport process is copied from the original version of the backport workflow from https://github.com/kumahq/.github/blob/main/.github/workflows/wfc_backport.yml